lib/src/wrappers/ResponsiveWrapper.tsx

```diff
@@ -10,8 +10,8 @@ export interface ResponsiveWrapperProps
   extends DesktopWrapperProps,
     DesktopPopperWrapperProps,
     MobileWrapperProps {
-  /** Breakpoint when `Desktop` mode will be changed to `Mobile`
-   * @default 'md'
+  /** Breakpoint when `Mobile` mode will be changed to `Desktop`
+   * @default 'sm'
    */
```

Comment:
```suggestion
  /**
   *  The desktop version will be shown instead of the mobile one for any screen width over this breakpoint.
   * @default 'sm'
   */
```